### PR TITLE
Fix #57: Add option for fuzzy matching.

### LINKED ISSUE #57: Leverage fuzzaldrin like fuzzy-finder
Often, I'm in the situation where the directory listing looks like:
```
...
0092-name-lowering.txt
0093-global-value-numbering.txt
0094-loop-fission.txt
...
```

I have to switch to my terminal, and `atom *global-value-numbering*`. We can leverage fuzzaldrin quite easily to ease the pain.

### PR DESCRIPTION
**Pull request by Osmose**
@artagnon @jeffgran @JesseLeite If ya'll could pull down this branch and test it out locally, your feedback on how it feels would be greatly appreciated. :D

### CODE REVIEW

**Comment by jeffgran** [comment]
just tried it, love it. Thanks for getting this working. :)

**Comment by Osmose** [comment]
Cool, thanks! This should be release sometime within the next week at the latest, I think.

**Comment by artagnon** [comment]
Currently on vacation, but tried it out on the flight. Works fine, but here are some future directions:
1. Find out how fuzzy-finder highlights matching characters, and replicate it.
2. When attempting to prefix some existing text, the cursor jumps to the end of the line on every keystroke. This is a bug.
3. Ctrl-i/k should probably be deprecated. Ctrl-K is kill-line in core, and should remain that way.

**Comment by Osmose** [comment]
> 2. When attempting to prefix some existing text, the cursor jumps to the end of the line on every keystroke. This is a bug.

That's #84 and should be fixed in the latest release. Are you still hitting it?

**Comment by vdun** [comment]
What about highlighting ?

**Comment by Osmose** [comment]
> What about highlighting ?

I'm not sure I understand what you're asking. Can you rephrase?

**Comment by vdun** [comment]
I mean highlighting the typed characters.
Example below how charachters are highlighted in 'bold white' in the command palette :
highlighting

**Comment by Osmose** [comment]
Ah. You should file a new issue for that. At a glance I'd probably accept a PR for it.

**Comment by vdun** [comment]
Done: #147
